carbonmark-api/test/fixtures/digitalCarbon.ts

```diff
@@ -59,7 +59,19 @@ export const digitalCarbonProject = aCarbonProject({
   carbonCredits: [carbonCredit as CarbonCredit],
 });
 
+const empty_countries = {
+  data: {
+    carbonProjects: [],
+  },
+};
+
+export const token = {
```

Comment:
This might be better off in the `fixtures.tokens.ts` file

Reply:
Done 👍